test-tests/shared/src/main/scala/zio/test/GenUtils.scala

```diff
@@ -1,10 +1,11 @@
 package zio.test
 
-import scala.concurrent.Future
-
-import zio.{ DefaultRuntime, Managed, UIO, ZIO }
+import zio.Exit.{ Failure, Success }
 import zio.random.Random
 import zio.test.environment.TestRandom
+import zio._
+
+import scala.concurrent.Future
 
 object GenUtils extends DefaultRuntime {
```

Comment:
Can we remove the `DefaultRuntime` here?

Reply:
`GenSpec` still use it

Reply:
Good point.